src/library/dlgtrackinfo.cpp

```diff
@@ -278,7 +278,7 @@ void DlgTrackInfo::populateCues(TrackPointer pTrack) {
     while (it.hasNext()) {
         CuePointer pCue = it.next();
         Cue::CueType type = pCue->getType();
-        if (type == Cue::CUE || type == Cue::INTRO || type == Cue::OUTRO) {
+        if (type == Cue::CUE || type == Cue::INTRO || type == Cue::OUTRO || Cue::LOOP) {
```

Comment:
This line should read:
```suggestion
        if (type == Cue::CUE || type == Cue::INTRO || type == Cue::OUTRO || type == Cue::LOOP) {
```

Reply:
Thanks, I had already fixed that but `git revert` reintroduced the issue :(